korman/operators/op_mesh.py

```diff
@@ -100,15 +100,15 @@ def create_flare_objects(self):
         flare_root.plasma_modifiers.viewfacemod.preset_options = "Sprite"
 
         # Create a textured Plane
-        with utils.bmesh_object("{}_Visible".format(self.name_stem)) as (flare_plane, bm):
-            flare_plane.hide_render = True
-            flare_plane.plasma_object.enabled = True
-            bpyscene.objects.active = flare_plane
-
+        flare_plane = utils.BMeshObject(f"{self.name_stem}_Visible", managed=False)
+        flare_plane.hide_render = True
+        flare_plane.plasma_object.enabled = True
+        bpyscene.objects.active = flare_plane
```

Comment:
> TypeError: BPy_PropertyRNA - Attribute (setattr): SceneObjects.active expected a Object type, not BMeshObject

Reply:
```suggestion
        bpyscene.objects.active = flare_plane.object
```

Reply:
This suggestion fixes the export error, but the flare is still not visible.

Reply:
L#120 & L#123 also require flare_plane.object.  Alas, I cannot leave suggestions on unedited lines.